pkg/machine/wsl/util_windows.go

```diff
@@ -148,7 +115,7 @@ func relaunchElevatedWait() error {
 		lpFile:       uintptr(unsafe.Pointer(exe)),
 		lpParameters: uintptr(unsafe.Pointer(arg)),
 		lpDirectory:  uintptr(unsafe.Pointer(cwd)),
-		nShow:        1,
+		nShow:        syscall.SW_HIDE,
```

Comment:
I think the previous approach is problematic because the window should not be displayed. Therefore, I have changed it here to be hidden.

Reply:
It was intended to be displayed since it is executing the same operation which itself is displayed

Reply:
If we forward the stdout/stderr/stdin of the child process (with administrator privileges) to the parent process (with normal privileges), can we hide the window of the child process?

Code similar to this:

```go
var kernel32 := windows.NewLazySystemDLL("kernel32.dll")
var freeConsole = kernel32.NewProc("FreeConsole")
var attachConsole = kernel32.NewProc("AttachConsole")

func FreeConsole() error {
	if ret, _, lastErr := freeConsole.Call(); ret == 0 {
		return lastErr
	}

	return nil
}

func AttachConsole(pid uintptr) error {
	if ret, _, lastErr := attachConsole.Call(pid); ret == 0 {
		return lastErr
	}

	return nil
}

const ATTACH_PARENT_PROCESS = ^uintptr(0)

// MoveConsoleToParent moves the console(stdout / stderr / stdin) to the parent process
func MoveConsoleToParent() error {
	if err := winapi.FreeConsole(); err != nil {
		return fmt.Errorf("failed to free console: %v", err)
	}

	if err := winapi.AttachConsole(ATTACH_PARENT_PROCESS); err != nil {
		return fmt.Errorf("failed to attach console: %v", err)
	}

	//  Update the standard handles in the `syscall` package
	// https://github.com/golang/go/blob/go1.20.5/src/syscall/syscall_windows.go#L493-L495
	syscall.Stdin, _ = syscall.GetStdHandle(syscall.STD_INPUT_HANDLE)
	syscall.Stdout, _ = syscall.GetStdHandle(syscall.STD_OUTPUT_HANDLE)
	syscall.Stderr, _ = syscall.GetStdHandle(syscall.STD_ERROR_HANDLE)

	// Update the corresponding file objects in the `os` package
	// See: https://github.com/golang/go/blob/go1.20.5/src/os/file.go#L65-L67
	os.Stdin = os.NewFile(uintptr(syscall.Stdin), "/dev/stdin")
	os.Stdout = os.NewFile(uintptr(syscall.Stdout), "/dev/stdout")
	os.Stderr = os.NewFile(uintptr(syscall.Stderr), "/dev/stderr")

	return nil
}
```

> This is just a simple technical discussion. I can accept whichever method is eventually chosen :)

Reply:
Sure. I don't see a problem with that.

Reply:
Considering that these features might be removed in the future, they will be maintained as-is, without adding code complexity for user experience.